oauth2-server/src/test/java/com/clouway/oauth2/IssueNewTokenForClientTest.java

```diff
@@ -94,20 +94,13 @@ public void tokenCannotBeIssued() throws Exception {
 
       oneOf(identityFinder).findIdentity(with(any(String.class)), with(any(GrantType.class)), with(any(DateTime.class)));
       will(returnValue(Optional.of(identity)));
-
```

Comment:
The second comment is not going to be addressed here unless you see it as reasonable to be here.